testground/benchmark/benchmark/stats.py

```diff
@@ -57,6 +57,7 @@ def dump_block_stats(
         blocks.append((txs, timestamp))
         tps = calculate_tps(blocks[-TPS_WINDOW:])
         tps_list.append(tps)
+        print("block", i, txs, timestamp, tps)
```

Comment:
can we search old logs from the dashboard?

Reply:
I think so, and log is kept much longer

Reply:
another issue is the `fp` passed can be stdout already, in that case we'll duplicate it.